### PR TITLE
[New][author] Add TheoremJS w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by arguiot**
# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/arguiot/TheoremJS
* Official website (optional, not the repository): https://theorem.js.org
* NPM package url (optional): https://www.npmjs.com/package/theorem.js
* License and its reference: MIT License
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 9
  * Count of stars: 527
  * Count of forks: 15
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by arguiot** [comment]
@dakshshah96 @sufuf3 I don't understand what's happening. The drone ci is telling me that the filename provided is wrong, is there any way to visualize / access the tree of `./ajax/libs/TheoremJS/1.1.0/` so I can fix the filename?

File: ajax/libs/TheoremJS/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "TheoremJS",
+  "filename": "dist/theorem.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/TheoremJS/package.json]: `dist` in `filename` should be removed

**Comment by arguiot** [comment]
@PeterDaveHello I fixed everything, thank for your support.

**Comment by PeterDaveHello** [comment]
@arguiot thank you! We'll have it reviewed soon.

**Comment by extend1994** [comment]
BTW, would be great if v1.2.0 can be released on npm 😃

**Comment by arguiot** [comment]
@extend1994 it will 😊

**Comment by arguiot** [comment]
@extend1994 After some review, I deleted v1.2.0 on GitHub because it was causing some issues.
Everything should be good now.

**Comment by extend1994** [comment]
@arguiot Thanks for the update!

**Comment by arguiot** [comment]
@PeterDaveHello When do you think 🤔 this PR will be merged?

**Comment by arguiot** [comment]
Any updates?

**Comment by extend1994** [comment]
@arguiot I can only approved this PR, but don't have permission to merge it, only Peter can.
Guess he's busy this week, please be patient.

**Comment by arguiot** [comment]
@extend1994 no problem, I'll wait. I just wanted some updates 😊

**Comment by arguiot** [comment]
@PeterDaveHello I'm about to release v1.2.0 very soon, it would be extremely cool if you could review this PR so I can announce that TheoremJS is on CDNJS.

P.S. thank you in advance 😊

**Comment by arguiot** [comment]
Any update?

**Comment by arguiot** [comment]
@PeterDaveHello Will this PR be merged any time soon?

**Comment by sashberd** [comment]
@arguiot As you can see @PeterDaveHello is not active more then month in any of its project. So do not build illusion about PR merging in short time cause only he has permission to merge it

**Comment by arguiot** [comment]
@sashberd Thank you very much for answering.

**Comment by extend1994** [comment]
Hi @arguiot 
I am authorized to take care of this PR, so I can help merge it and import all versions.
Just wondering can you provide v0.0.1 ~ v0.1.0 minified js files? As I encountered "parsing error" when I tried to help minify `theorem.js` using https://github.com/PeterDaveHello/web-minify-helper.

**Comment by arguiot** [comment]
@extend1994 Thank you very much for taking care of this PR 😊.
It's normal you encountered errors with `v0.1.0` and `v0.0.1` as they were written in ES6. I compressed the files myself, and here they are in a ZIP format (because GitHub didn't allow me to upload them directly).

[Archive.zip](https://github.com/cdnjs/cdnjs/files/2453873/Archive.zip)

**Comment by extend1994** [comment]
@arguiot No problem :)
There are `theorem.min.js` and `theorem@0.0.1.min.js` in your provided archive,
not sure `theorem.min.js` is the minified file for v0.1.0?

Can you also provide one for v0.0.2? Thanks.

**Comment by arguiot** [comment]
@extend1994 I renamed it `theorem@0.1.0.min.js` for more readability. I also added `v0.0.2`. 
If there is anything else, don't hesitate to ask 😊.

[Archive.zip](https://github.com/cdnjs/cdnjs/files/2468694/Archive.zip)

**Pull request merged**